znc/content.md

```diff
@@ -0,0 +1,33 @@
+# What is ZNC?
+
+ZNC is an IRC network bouncer (BNC). It can detach the client from the
+actual IRC server, and also from selected channels. Multiple clients
+from different locations can connect to a single ZNC account
+simultaneously and therefore appear under the same nickname on IRC.
+
+[ZNC Wiki](http://znc.in/)
+
+# How to use this image
+
+ZNC in this image stores its configuration in `/znc-data`.
+If you have existing configuration, you can reuse it with `-v $HOME/.znc:/znc-data`.
+Alternatively, you can create a new config in a volume or in a local dir.
+The examples below assumes a volume named `znc-cfg`.
+
+```console
+$ docker run -it -v znc-cfg:/znc-data znc --makeconf
+```
+
+To run ZNC:
+
+```console
+$ docker run -p 6697:6697 -v znc-cfg:/znc-data znc
+```
+
+The port should match the port you used during `--makeconf`. Note that 6667 is
+often blocked by web browsers, and therefore is not recommended.
+
+If you use any external module, put the .cpp, .py or .pm file to
+`/znc-data/modules` (you may need to create that directory).
+
+IPv6 is currently disabled in this image.
```

Comment:
Do you think it'd be worthwhile to expound a little bit about why here, similar to the notes in the `Dockerfile` comments?